README.adoc

```diff
@@ -3,137 +3,359 @@
+
+==== Stage
+In order to have private maven releases, you can modify `RELEASE_GIT_REPOSITORY` or `MAVEN_REPOSITORY_NAME`, to respectively use code from a private git repository and then push artifacts to a private maven repository.
+
+Artifact promotion is done in the next stage 'packaging'.
+
+=== Packaging
+
+The packaging process looks after the latest Jenkins version published on a Maven repository and then build and publish artifacts for Debian, Red Hat, SUSE, Windows. It also republishes the War file on the package server.
+If enabled, then it promotes git commits between git repository, promotes maven artifacts between maven repository.
+
+NOTE: Packages are not re-published if they already exist, only package website is overridden so it's safe to re-trigger the job.
+
+
+==== Steps
 
 Estimated time +- 30min
 
 . Connect to the Jenkins VPN
 . Open your favorite browser to https://release.ci.jenkins.io[release.ci.jenkins.io]
-. Trigger the packaging job on the master branch. https://release.ci.jenkins.io/blue/organizations/jenkins/core-package/branches[Link]
+. Trigger the packaging job on the master branch. https://release.ci.jenkins.io/blue/organizations/jenkins/core%2Fpackage/branches/[Link]
 . Once triggered, it asks you which release line you want to package for. The release line matches one of the profile defines in https://github.com/jenkins-infra/release/tree/master/profile.d[profile.d], so please carefully review those settings in order to validate that's what you are looking for.
 
-Once the job is done, every package will be published and then mirror synced
+Once the job is done, every package will be published and then mirror synchronized.
 
-=== Validate
+==== Validate
 Ensure that packages are correctly published on pkg.jenkins.io and correctly signed.
 
-=== Stage
-Staging packages is not supported.
+==== Stage
+Staging packages is not yet fully supported, more information on link:https://issues.jenkins-ci.org/browse/INFRA-1363[INFRA-1363] and link:https://issues.jenkins-ci.org/browse/INFRA-2608[INFRA-2608]
+
+== Profile
+The release profile is used to identify the kind of release we are going to do. They are mainly influenced by following elements:
+
+. Do we want to releases based on different repository branch?
+. Do we want to release based on different git repository?
+
+At the moment we identify four release type
+
+. Weekly
+. Stable
+. Security
+. Lts security
+
+[NOTE]
+====
+link:https://release.ci.jenkins.io[release.ci.jenkins.io] is designed in a way that we have two generic jobs, one for release and a second one for packaging.
+Then we have one job per release type which triggered the two generic Jenkins jobs with different parameters.
+While it isn't required to trigger a release type job, it just increase visibility and reduce human error factor.
```

Comment:
I'm not sure where I can find the trigger job.  Could we review that together in one of our training sessions?
```suggestion
link:https://release.ci.jenkins.io[release.ci.jenkins.io] has two generic jobs, one for release and a second one for packaging.
One job per release type triggers the two generic jobs with different parameters.
While it isn't required to trigger a release type job, it increases visibility and reduces the risk of human error.
```